ray-operator/Makefile

```diff
@@ -58,7 +60,13 @@ test: manifests generate fmt vet ## Run tests.
 ##@ Build
 
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	go build                                    \
```

Comment:
minor: Can we format `\` ?

Reply:
![image](https://user-images.githubusercontent.com/4739316/156270188-cce00828-bee3-4b62-8d6a-efa4de45c49a.png)

Reply:
Sorray, I have formatted it.